lib/tasks/migrate_site_banners.rake

```diff
@@ -0,0 +1,18 @@
+namespace :site_banners do
+  task :migrate_site_banners => :environment do
+    # Update record_id's to use uuids instead of bigints
+    SiteBanner.all.each do |site|
```

Comment:
Actually you should iterate over ActiveStorage::Attachment because there's 1 logo and 1 background per SiteBanner.

Reply:
Done ✔️